src/components/calculator.js

```diff
@@ -1,29 +1,50 @@
+
     return (
       <div className="calculator">
-        <input type="text" id="entry" className="calc" name="entry" placeholder="0" />
-        <div className="calc btn">AC</div>
-        <div className="calc btn">+/-</div>
-        <div className="calc btn">%</div>
-        <div className="calc btn orange">/</div>
-        <div className="calc btn">7</div>
-        <div className="calc btn">8</div>
-        <div className="calc btn">9</div>
-        <div className="calc btn orange">*</div>
-        <div className="calc btn">4</div>
-        <div className="calc btn">5</div>
-        <div className="calc btn">6</div>
-        <div className="calc btn orange">-</div>
-        <div className="calc btn">1</div>
-        <div className="calc btn">2</div>
-        <div className="calc btn">3</div>
-        <div className="calc btn orange">+</div>
-        <div id="zero" className="calc btn">0</div>
-        <div className="calc btn">.</div>
-        <div className="calc btn orange">=</div>
+        <div id="entry" className="calc" name="entry" placeholder="0">
+          <span>{total}</span>
+          <span>{operation}</span>
+          <span>{next}</span>
+        </div>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>AC</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>+/-</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>%</button>
+        <button type="button" className="calc btn orange" onClick={this.clickHandle}>÷</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>7</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>8</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>9</button>
+        <button type="button" className="calc btn orange" onClick={this.clickHandle}>x</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>4</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>5</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>6</button>
+        <button type="button" className="calc btn orange" onClick={this.clickHandle}>-</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>1</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>2</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>3</button>
+        <button type="button" className="calc btn orange" onClick={this.clickHandle}>+</button>
+        <button type="button" id="zero" className="calc btn" onClick={this.clickHandle}>0</button>
+        <button type="button" className="calc btn" onClick={this.clickHandle}>.</button>
+        <button type="button" className="calc btn orange" onClick={this.clickHandle}>=</button>
```

Comment:
**Changes Required**
- [ ] The button can be extracted as it's own Class Component. Please let us store the button names into an array and use `Array.map` to display then pro grammatically. This will allow us to stick to the `DRY. principle.

Reply:
Required changes satisfied.